airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -522,9 +522,13 @@ def _sync_appbuilder_roles(self):
         # Otherwise, when the name of a view or menu is changed, the framework
         # will add the new Views and Menus names to the backend, but will not
         # delete the old ones.
-        if conf.getboolean(
-            "fab", "UPDATE_FAB_PERMS", fallback=conf.getboolean("webserver", "UPDATE_FAB_PERMS")
-        ):
+        from tests.test_utils.compat import AIRFLOW_V_3_0_PLUS
```

Comment:
This imports a test module in a provider…?? Does not look right to me.

Reply:
Yep, you can also do it like this:

```
from airflow.version import version

  if Version(version) >= Version("3.0"):
```

Reply:
My bad. It's resolved.